README.md

```diff
@@ -118,6 +125,26 @@ end
 
 - `value` - The value to set for the given option. Required for the set action. Booleans are mapped to `on` or `off`.
 
+### gluster\_mountbroker\_user
+
+Use this resource to allow or disallow the given user access to the given volume:
+
+```ruby
+gluster_mountbroker_user 'user/volume_name' do
+  action :add
+end
+```
+
+```ruby
+gluster_mountbroker_user 'user/volume_name' do
+  action :remove
+end
+```
+
+#### Parameters
+
+None.
```

Comment:
That only triggers now if the two resources conflict in some way, not sure of the details but I've barely seen that warning lately after seeing it loads before. If it is triggered because of a conflict then I think the user needs to remove one of the resources rather than just rename one of them. It doesn't make sense to set an option to some value and then set it to some other value in the same run. Similarly, it doesn't make sense to add permission for some user and then remove it again in the same run.